scala/scala.bzl

```diff
@@ -440,7 +441,8 @@ def scala_test_suite(name,
         visibility = visibility,
         size = size,
         colors = colors,
-        full_stacktraces = full_stacktraces)
+        full_stacktraces = full_stacktraces,
+        unused_dependency_checker_mode = "off")
```

Comment:
Apologies for commenting on an ancient PR, but I just dug through history do understand this. Am I right in my understanding that for `_suite`-like rules the dependency checker must be disabled as each individual "sub-target" might not depend on the whole set of dependencies for the overall build target? Is there a way in which dependency checking can be done at the target level while being disabled for "sub-target"s? (also apologies if I'm not using the correct wording, I hope that's understandable)

Reply:
I think indeed suite like rules weren't in the focus of the design. I suggest opening a new issue and referencing this PR to discuss this with current maintainers

Reply:
Thanks!